www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -23,224 +23,282 @@ description: How to manage platforms and Cordova CLI versions.
 
 ```bash
-"cordova": {"plugins": ["cordova-plugin-console"]},"dependencies": {"cordova-plugin-console": "^1.0.0"}
+cordova plugin add <plugin[@<version>] | directory | git_url> --nosave
 ```
 
-The '--nosave' flag prevents adding and deleting specified plugins from config.xml and package.json. To prevent saving a plugin, you issue the following command :
+Some Examples:
 
-```bash
-$ cordova plugin add <plugin[@<version>] | directory | git_url> --nosave
-```
+* **`cordova plugin add cordova-plugin-device`**
+
+  Retrieves the pinned version of the `cordova-plugin-device` plugin from npm, adds it to the project and updates the `package.json` file.
+
+* **`cordova plugin add cordova-plugin-device@2.0.1`**
+
+  Retrieves the `cordova-plugin-device` plugin at version `2.0.1` from npm, adds it to the project and updates the `package.json` file.
+
+* **`cordova plugin add https://github.com/apache/cordova-plugin-device.git`**
+  
+  **`cordova plugin add https://github.com/apache/cordova-plugin-device`**
+  
+  **`cordova plugin add github:apache/cordova-plugin-device`**
 
-Some examples :
+  npm retrieves the `cordova-plugin-device` plugin from the git repository, adds it to the project and updates the `package.json`.
 
-  * **'cordova plugin add cordova-plugin-console'** => retrieves the pinned version of the console plugin, adds it to the project and then updates config.xml and package.json.
-  * **'cordova plugin add cordova-plugin-console@0.2.13'** => retrieves the android plugin, version 0.2.13 from npm, adds it to the project and then updates config.xml and package.json.
-  * **'cordova plugin add https://github.com/apache/cordova-plugin-console.git'** => npm installs specified console plugin from the git repository, adds the console plugin to the project, then updates config.xml and and package.json and points its version to the specified git-url.
-  * **'cordova plugin add C:/path/to/console/plugin'** => retrieves the console plugin from the specified directory, adds it to the project, then updates config.xml and package.json and points to the directory.
+* **`cordova plugin add C:/path/to/console/plugin`**
 
-### Mass saving plugins on an existing project
-If you have a pre-existing project and you want to save all currently added plugins in the project, you can use :
+  Retrieves the `cordova-plugin-device` plugin from the specified directory, adds it to the project, and updates the `package.json` file.
+
+### Mass Saving of Plugins on an Existing Project
+
+If you have a pre-existing project and you want to save all currently added plugins in the project, you can use:
 
 ```bash
-$ cordova plugin save
+cordova plugin save
 ```
 
+### Removing Plugins
 
-### Removing plugins
-It is also possible to delete from config.xml and package.json during the command 'cordova plugin remove' :
+It is also possible to delete from `config.xml` and `package.json` during the command `cordova plugin remove`:
```

Comment:
"also possible"? What does this refer to?

Reply:
The person who wrote this originally was stream lining from the previous section. The previous section talks about adding plugins so therefor it is also possible to remove.

I went ahead removed `also`, among other rewording changes.